.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,40 @@
-<!-- Please replace #XX below with an existing issue number. Remove the line entirely if none exist. -->
-Fixes #XX
+## Fixes
+<!-- If PR only partly solves an issue, replace 'Fixes' below with 'Partially addresses'. -->
+<!-- If there is no issue being resolved, open one. -->
+Fixes #[issue number] by @[issue author]
 
 ## Description
-<!-- Add your description below. -->
+<!-- Give a clear and concise description of what the pull request does. -->
 
-## Other information
-<!-- Add any other information below or delete the "Other Information" line entirely. -->
+## Other information or technicalities
+<!-- Add any other information or technical details about the implementation; or delete the section entirely. -->
+
+## Tests
+<!-- Give steps for the reviewer to verify that this PR fixes the problem; or delete the section entirely. -->
+
+## Screenshots
+<!-- Add screenshots to show the problem and the solution; or delete the section entirely. -->
 
 ## Checklist
+<!-- Replace  the [ ] with [x] to check the boxes. --> 
 - [ ] My pull request has a descriptive title (not a vague title like `Update
   index.md`).
 - [ ] My pull request targets the `master` branch of the repository.
 - [ ] My commit messages follow [best practices][best_practices].
+- [ ] My code follows the established code style of the repository.
```

Comment:
This is really not something I expect a contributor to assess.  It makes more sense as a guideline to contributing before the fact, not a last-minute check box.

Reply:
I definitely think this makes sense in the contribution guidelines. I think we should be testing code style through automated linting, not through a check box.

Reply:
In that case, I'd like to point out the checkbox right above this one. Wouldn't it be too late to evaluate commit messages by the time one is opening a PR? In my view, the checkbox's purspose was just to ensure the contributor did those checks beforehand and dissuade them from opening the PR just in case they forgot.

Reply:
In that case, I'd like to point out the checkbox right above this one. Wouldn't it be too late to evaluate commit messages by the time one is opening a PR? In my view, the checkbox's purspose was just to ensure the contributor did those checks beforehand and dissuade them from opening the PR just in case they forgot.

Reply:
That's true, let's leave it in.